ApartApps.md

```diff
@@ -0,0 +1,107 @@
+- They define which variables correspond to which columns to be inserted into or updated in an `INSERT` or `UPDATE` stored procedure.
+
+All view models related to the apartment application begin with the word `Apartment`.
+
+### Database Tables
+
+Data specifically intended for the apartment application is stored in the CCT database in a series of tables whose names begin with `AA_`.
+
+The details of these tables can be viewed and edited in SQL Server Management Studio with the right permissions. 
+Although, a faster method for simply viewing the tables is to open `Gordon360/Models/CCT_DB_Models.edmx` in Visual Studio 2017 and pan the screen around until the box for the desired table is in view. In the edmx, the type and nullability of each attribute of a table can also be viewed by clicking a column name and looking at the Properties table in the bottom right corner of Visual Studio 2017.
+
+
+### Stored Procedures
+
+Stored Procedures are SQL scripts that are stored in the database. As it is with most of the 360 site’s features, the apartment application stored procedures can be found in `CCT/Programmability/Stored_Procedures/`. All the ones related to the apartment application have the acronym `AA` in their name after the word indicating what type of procedure they are (INSERT_AA_..., GET_AA_…, etc.). Each procedure’s description and code can be viewed by right-clicking its name and choosing `Modify` from the menu.
+
+
+## Where to Find the Code for X
+
+Note: This part of the documentation is the most vulnerable to becoming outdated! 
+
+- __The route to call in order to perform a function in the backend__ 
+  - In other words, what would X be here in your frontend code: `result = await http.get(`housing/X/`);` 
+  - Just look above the declaration of the Housing Controller method you want to call to find something of the form `[Route("X")]` 
+  - Just be careful that the the line above that, which is either `[HttpGet]`, `[HttpPut]`, `[HttpPost]`, or `[HttpDelete]`, corresponds to the type of operation you want to perform
+  - If part of X has `{` and `}` around it, that is a parameter, and you have to do `result = await http.get(`housing/someRoute/${valuePassingIn}`);`
+- __The identity and data type of any parameters that must be passed from the frontend__
+  - There are two kinds of parameters that the frontend must supply, those that should be named in the route and those that should be placed in the body of the HTTP request 
+  - Those that should be named in the route are identified in the same place as the previous point, `[Route("someRoute/{X}")]` in the respective Housing Controller method
+  - The data type for these, let's call it Y, is identified next to the corresponding parameter in the actual method declaration: `public IHttpActionResult DoSomething(Y X)` 
```

Comment:
```suggestion
  - The data type for these, let's call it Y, is identified next to the corresponding parameter in the actual method declaration: `public IHttpActionResult DoSomething(Y X)` 
    - Since these "inline" parameters are passed in the form of text that is part of the URL, the data type is limited to primitive types, such as `string` or `int` for example.
```